src/traversals/greedy_color.jl

```diff
@@ -23,16 +23,18 @@ function perm_greedy_color(g::AbstractGraph, seq::Vector{T}) where {T<:Integer}
 
     has_self_loops(g) && throw(ArgumentError("graph must not have self loops"))
 
+    colors_used = BitSet()
```

Comment:
```suggestion
    colors_used = Set{T}()
```
In most situations of interest the graph will be sparse, so the number of colors among the neighbors will be negligible compared to the total number of colors. This makes `Set` more efficient than `BitSet` asymptotically.